runtime/linux/src/main/java/org/qbicc/runtime/linux/SysSyscall.java

```diff
@@ -0,0 +1,14 @@
+package org.qbicc.runtime.linux;
+
+import static org.qbicc.runtime.CNative.*;
+
+/**
+ *
+ */
+@include("<sys/syscall.h>")
+public class SysSyscall {
+    public static native c_long syscall(c_long number, object... args);
+
+    public static final c_long SYS_futex = constant();
+    public static final c_long SYS_futex_time64 = constant();
```

Comment:
why do we need `SYS_futex_time64` constant, doesn't seem to be used anywhere.

Reply:
I implemented these constants first, before I knew what I was going to need. I could remove it.